shared/test/middleware/helpers/test_storage_id.rb

```diff
@@ -0,0 +1,75 @@
+require_relative '../../test_helper'
+require_relative '../../../middleware/helpers/storage_id'
+
+class StorageIdTest < Minitest::Test
+  def setup
+    @user_storage_ids_table = PEGASUS_DB[:user_storage_ids]
+    @initial_num_entries = @user_storage_ids_table.count
+  end
+
+  def teardown
+    num_entries = @user_storage_ids_table.count
+    @user_storage_ids_table.order(:id).last(num_entries - @initial_num_entries).each do |entry|
+      @user_storage_ids_table.where(id: entry[:id]).delete
+    end
+  end
```

Comment:
Not sure if there's a better way to do this. I expected to be able to do `@user_storage_ids_table.order(:id).last(num_entries - @initial_num_entries).delete` but that gave me syntax errors.

Reply:
Odd. I'd also expect [delete](http://sequel.jeremyevans.net/rdoc/classes/Sequel/Dataset.html#method-i-delete) to work. I wonder if it may be the inclusion of [last](http://sequel.jeremyevans.net/rdoc/classes/Sequel/Dataset.html#method-i-last), though presumably that just adds a `LIMIT` to the query which should be compatible with `DELETE`. It could be worth trying with the specific set of ids rather than an order by and limit? It's also not that important.

Reply:
Ignore this for now, I think i found a better way of doing this:
```
# Use this class as the base class for your tests
class TransactionalMinitest < Minitest::Test
  def run(*args, &block)
    Sequel::Model.db.transaction(:rollback=>:always, :auto_savepoint=>true){super}
  end
end

Reply:
Are we sure that all the last n storage ids are from this test? Is it possible other tests could run in parallel and interfere? Would it make sense instead to collect the inserted ids in the test (below) and delete those in teardown?

Reply:
It might be worth making a similar base class to https://github.com/code-dot-org/code-dot-org/blob/65a46d3b8cb8eb616dc8fb5e5ab63eeebf80bcee/pegasus/test/sequel_test_case.rb, or moving that one to shared